src/styles.scss

```diff
@@ -1475,6 +1476,23 @@ hr {
   font-size: 12px;
 }
 
+
+.search-form-field {
+  max-width: 200px;
+}
+
+.search-form-field input {
+  padding-right: 35px;
```

Comment:
if you're finding that the x is disappearing without the padding-right, it's likely that the width of the search-form-field is wrong

Reply:
I added the padding so when a long input is passed it doesn't cover the x icon

Without padding:
<img width="232" alt="Screen Shot 2021-05-28 at 4 55 37 PM" src="https://user-images.githubusercontent.com/65895867/120040992-8974bb00-bfd5-11eb-9ef4-3ce10f2134d0.png">

Reply:
With 0 styling: 
![image](https://user-images.githubusercontent.com/24548904/120042550-49630780-bfd8-11eb-9079-cb2bef13d4d4.png)